libs/platform/src/lib/components/dynamic-page/dynamic-page.component.html

```diff
@@ -0,0 +1,37 @@
+<section class="fd-dynamic-page" [attr.aria-label]="ariaLabel">
+    <header style="left: 0; right: 0; z-index: 3; position: fixed;" class="fd-dynamic-page__header" #header>
+        <div>
+            <div (click)="toggleCollapse()">
+                <ng-content select="fdp-dynamic-page-title"></ng-content>
+            </div>
+            <ng-content select="fdp-dynamic-page-header"></ng-content>
+        </div>
+    </header>
+
+    <div
+        class="fd-dynamic-page__tabs--overflow"
+        *ngIf="isTabbed"
+        style="left: 0; right: 0; position: relative; top: 0;"
```

Comment:
Could you change inline styling to some custom class?

Reply:
added to a custom class now